tests/server.go

```diff
@@ -0,0 +1,49 @@
+package tests
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"kubevirt.io/client-go/kubecli"
+)
+
+func NewHTTPServerPod(port int) *corev1.Pod {
+	serverCommand := fmt.Sprintf(`echo -e "HTTP/1.1 200 OK\nContent-Length: 11\n\nHello World!" | nc -kvlp %d`, port)
+	return RenderPod("http-hello-world-server", []string{"/bin/bash"}, []string{"-c", serverCommand})
+}
+
+func NewTCPServerPod(port int) *corev1.Pod {
+	serverCommand := fmt.Sprintf("nc -klp %d --sh-exec 'echo \"Hello World!\"'", port)
+	return RenderPod("tcp-hello-world-server", []string{"/bin/bash"}, []string{"-c", serverCommand})
```

Comment:
Relying on `RenderPod` will get you a pod running containers whose image is VM killer, which is based on [kuebevirt-testing](https://github.com/kubevirt/kubevirtci/blob/master/cluster-provision/images/kubevirt-testing/Dockerfile) image.

Cutting edge fedora 28 . I don't think we should use that anywhere. I'll work a way to build on renderPod so you have a say on the image you create.